SampleApp/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,1899 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is how Cocoapods generates it so yes. Note this is the Pods workspace, not the Sample App workspace.